lib/helpers/answerHelper.js

```diff
@@ -10,9 +10,7 @@ function dateFormatter(date) {
 }
 
 function isValidDate(str) {
-    const d = Date.parse(str);
-    // eslint-disable-next-line no-restricted-globals
-    return !isNaN(d);
+    return moment(str, 'YYYY-MM-DD[T]HH:mm:ss.SSS[Z]', true).isValid();
```

Comment:
I'm not sure if you tried it, but I think moment has a constant for ISO 8601 date formats e.g. moment.ISO_8601

http://momentjs.com/docs/#/parsing/special-formats/

Might be worth sticking with its own implementation?

Reply:
@sinclairs I was aware of the constant. I'd need to test further, but I don't think it was behaving as expected, and this was a work around I found. Soemthing to do with this i believe might have been causing the issue: 
[(https://github.com/moment/moment/issues/3019)](https://github.com/moment/moment/issues/3019)

Reply:
Ah, ok, no probs. We'll go with your working solution.